_includes/configuration.md

```diff
@@ -81,18 +91,32 @@ necessary to alter the `max_connections` setting.
 
 4.  Select **Configure Application** from the menu.
 
-5.  Choose **Create Region in External Database** for the database location.
+5.  You are prompted to create or fetch a security key.
```

Comment:
What prompted adding this?  I'm not against it, per se, but it requires more of a review from me for accuracy, and I don't recall needing this.  If there was a private chat about it, let's discuss privately.